ports/rp2/mphalport.h

```diff
@@ -56,7 +60,21 @@ static inline mp_uint_t mp_hal_ticks_ms(void) {
 static inline mp_uint_t mp_hal_ticks_cpu(void) {
     // ticks_cpu() is defined as using the highest-resolution timing source
     // in the system. This is usually a CPU clock, but doesn't have to be.
-    return time_us_32();
+    return SYSTICK_MAX - systick_hw->cvr;
```

Comment:
Unfortunately this won't work, `time.ticks_cpu()` will no longer handle overflow/wrap around correctly.  That's because `time.ticks_diff()` requires the wrapping to be at 32-bits.

Reply:
That's right. So I drop this change. It is not required for rp2_bitstream.